### PR TITLE
profile link added for ihtisham

### PR DESCRIPTION
**Pull request by ihtishamKhan**
I have added my profile link also some missing icons that I needed

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

The package.json file is added to the commit, it is because of my code editor added some tab space

Also, I have added a couple of missing icons that I needed

I was not able to `npm start` it was giving an error so I have to modify the script like: `"start": "react-scripts --openssl-legacy-provider start",` which I removed before commit

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1261"></a>

### CODE REVIEW

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey! please revert changes in  `package-lock.json`, `package-lock.json.`.

Also, create a separate PR for adding Icons.

**Comment by ihtishamKhan** [comment]
Hey! can someone guide me what is the correct way to revert these changes in the `package.json` and `package-lock.json` files?

Should I delete this PR and submit another one?

File: public/data/ihtishamKhan.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      "name": "Connect with me on LinkedIn",
+      "url": "https://www.linkedin.com/in/ihtisham-khan/",
+      "icon": "linkedin"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Worked on college project",
+      "date": "2019-2020",
+      "icon": "graduation-cap",
```
Comment (kumarsonsoff3) [inline public/data/ihtishamKhan.json:27]: `graduation-cap` is an incorrect keyword, use `graduation-hat` instead! 

```suggestion
      "icon": "graduation-hat",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Comment by kumarsonsoff3** [comment]
Also, I suggest you to use other icons in place of the **language icons** you have used until we add those icons to our [icons.md](https://github.com/EddieHubCommunity/LinkFree/blob/main/icons.md).

It would be great if you create an issue for adding new icons to linkfree, and create a PR with reference to that issue.
<hr/>

**What do other Reviewers think in this regard??**

**Review by emmalearnscode — APPROVED** [review]

**Comment by emmalearnscode** [comment]
> Also, I suggest you to use other icons in place of the **language icons** you have used until we add those icons to our [icons.md](https://github.com/EddieHubCommunity/LinkFree/blob/main/icons.md).
> 
> It would be great if you create an issue for adding new icons to linkfree, and create a PR with reference to that issue.
> 
> **What do other Reviewers think in this regard??**

We have a default icon in case the icons don’t yet exist and since a new issue has been raised I’d suggest leaving the icon names as they are. The default icon will be shown to start with but as soon as someone tackles the issue and adds the icons then they’ll be shown instead and no need for this file to be updated.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to me!

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

**Review by kumarsonsoff3 — APPROVED** [review]
LGTM 👍 Good to goo! 🚀

**Review by Panquesito7 — APPROVED** [review]

**Comment by ihtishamKhan** [comment]
@kumarsonsoff3 @emmalearnscode 
I can't see my code in the main branch, although it is merged

**Comment by emmalearnscode** [comment]
@ihtishamKhan merging doesn’t automatically deploy it, it’ll appear next time the site gets deployed

**Comment by ihtishamKhan** [comment]
@emmalearnscode just want to know that the file I have added is not even in the main branch, this one also to appear later when the site gets deployed?

**Comment by emmalearnscode** [comment]
@ihtishamKhan I can see it on the main branch under public/data (they’re alphabetical but small letters come after capitals so make sure you scroll down to i and not I )

**Comment by ihtishamKhan** [comment]
I didn't know about this type of sorting, Thank you so much @emmalearnscode

**Pull request merged**